#StalkRef.msg
#This message contains a reference input for the Giraff head actuators
Header header
float32 head_tilt
float32 head_height
